### PR TITLE
Improvement/05 triathlon

### PR DESCRIPTION
**Pull request by BlueEnni**
@TomasKostadinov please add the remaining html files to the combiner

triathlon/schwimmen/schwimmen.html
triathlon/schwimmen/trainingsplan-schwimmen.html
triathlon/laufen/laufen.html
triathlon/laufen/trainingsplan-laufen.html
triathlon/radfahren/radfahren.html
triathlon/radfahren/trainingsplan-radfahren.html
triathlon/mobi-stabi-training/mobi-stabi-training.html
triathlon/mobi-stabi-training/trainingsplan-mobi-stabi-training.html

### CODE REVIEW

File: site/triathlon/index.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        <h1>Sidebar</h1>
+        <div class="main-content">
+            <div class="sidebar">
+                <div class="sticky">
+                    <div class="description">
+                        <h3>Auf dieser Seite</h3>
+                    </div>
+                    <ul>
+                        <li class="active">
+                            <a href="index.html">Home</a>
```
Comment (TomasKostadinov) [inline site/triathlon/index.html]: Please don't use filenames. We later want to have clean urls like /triathlon (the server will find /triathlon/index.html).
Therefor also rename all files to index.html

File: site/triathlon/index.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+                        </li>
+                        <li>
+                            <a href="laufen/laufen.html">Laufen</a>
```
Comment (TomasKostadinov) [inline site/triathlon/index.html]: Change to /triathlon/laufen

File: site/triathlon/index.html [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+                        </li>
+                        <li>
+                            <a href="schwimmen/schwimmen.html">Schwimmen</a>
```
Comment (TomasKostadinov) [inline site/triathlon/index.html]: Change to /triathlon/schwimmen

File: site/triathlon/index.html [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+                        </li>
+                        <li>
+                            <a href="radfahren/radfahren.html">Radfahren</a>
```
Comment (TomasKostadinov) [inline site/triathlon/index.html]: Also here

File: site/triathlon/laufen/laufen.html [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+        <h1>Sidebar</h1>
+        <div class="main-content">
+            <div class="sidebar">
+                <div class="sticky">
+                    <div class="description">
+                        <h3>Auf dieser Seite</h3>
+                    </div>
+                    <ul>
+                        <li>
+                            <a href="../index.html">Home</a>
```
Comment (TomasKostadinov) [inline site/triathlon/laufen/laufen.html]: Change to /triathlon

File: site/triathlon/laufen/laufen.html [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+                        </li>
+                        <li class="active">
+                            <a href="laufen/laufen.html">Laufen</a>
```
Comment (TomasKostadinov) [inline site/triathlon/laufen/laufen.html]: Change this

File: site/triathlon/laufen/laufen.html [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+                        </li>
+                        <li>
+                            <a href="schwimmen/schwimmen.html">Schwimmen</a>
```
Comment (TomasKostadinov) [inline site/triathlon/laufen/laufen.html]: Change this

File: site/triathlon/laufen/laufen.html [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+                        </li>
+                        <li>
+                            <a href="radfahren/radfahren.html">Radfahren</a>
```
Comment (TomasKostadinov) [inline site/triathlon/laufen/laufen.html]: Change this

File: site/triathlon/laufen/trainingsplan-laufen.html [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+        <h1>Sidebar</h1>
+        <div class="main-content">
+            <div class="sidebar">
+                <div class="sticky">
+                    <div class="description">
+                        <h3>Auf dieser Seite</h3>
+                    </div>
+                    <ul>
+                        <li>
+                            <a href="../index.html">Home</a>
```
Comment (TomasKostadinov) [inline site/triathlon/laufen/trainingsplan-laufen.html]: Also here

File: site/triathlon/mobi-stabi-training/mobi-stabi-training.html [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+        <h1>Sidebar</h1>
+        <div class="main-content">
+            <div class="sidebar">
+                <div class="sticky">
+                    <div class="description">
+                        <h3>Auf dieser Seite</h3>
+                    </div>
+                    <ul>
+                        <li>
+                            <a href="../index.html">Home</a>
```
Comment (TomasKostadinov) [inline site/triathlon/mobi-stabi-training/mobi-stabi-training.html]: An here

File: site/triathlon/mobi-stabi-training/trainingsplan-mobi-stabi-training.html [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+        <h1>Sidebar</h1>
+        <div class="main-content">
+            <div class="sidebar">
+                <div class="sticky">
+                    <div class="description">
+                        <h3>Auf dieser Seite</h3>
+                    </div>
+                    <ul>
+                        <li>
+                            <a href="../index.html">Home</a>
```
Comment (TomasKostadinov) [inline site/triathlon/mobi-stabi-training/trainingsplan-mobi-stabi-training.html]: An here

File: site/triathlon/radfahren/radfahren.html [hunk 12]
```diff
@@ -0,0 +1,61 @@
...
+        <h1>Sidebar</h1>
+        <div class="main-content">
+            <div class="sidebar">
+                <div class="sticky">
+                    <div class="description">
+                        <h3>Auf dieser Seite</h3>
+                    </div>
+                    <ul>
+                        <li>
+                            <a href="../index.html">Home</a>
```
Comment (TomasKostadinov) [inline site/triathlon/radfahren/radfahren.html]: And here

File: site/triathlon/radfahren/trainingsplan-radfahren.html [hunk 13]
```diff
@@ -0,0 +1,59 @@
...
+        <h1>Sidebar</h1>
+        <div class="main-content">
+            <div class="sidebar">
+                <div class="sticky">
+                    <div class="description">
+                        <h3>Auf dieser Seite</h3>
+                    </div>
+                    <ul>
+                        <li>
+                            <a href="../index.html">Home</a>
```
Comment (TomasKostadinov) [inline site/triathlon/radfahren/trainingsplan-radfahren.html]: And here

File: site/triathlon/schwimmen/schwimmen.html [hunk 14]
```diff
@@ -0,0 +1,82 @@
...
+        <h1>Sidebar</h1>
+        <div class="main-content">
+            <div class="sidebar">
+                <div class="sticky">
+                    <div class="description">
+                        <h3>Auf dieser Seite</h3>
+                    </div>
+                    <ul>
+                        <li>
+                            <a href="../index.html">Home</a>
```
Comment (TomasKostadinov) [inline site/triathlon/schwimmen/schwimmen.html]: And here

File: site/triathlon/schwimmen/trainingsplan-schwimmen.html [hunk 15]
```diff
@@ -0,0 +1,105 @@
...
+        <h1>Sidebar</h1>
+        <div class="main-content">
+            <div class="sidebar">
+                <div class="sticky">
+                    <div class="description">
+                        <h3>Auf dieser Seite</h3>
+                    </div>
+                    <ul>
+                        <li>
+                            <a href="../index.html">Home</a>
```
Comment (TomasKostadinov) [inline site/triathlon/schwimmen/trainingsplan-schwimmen.html]: And here

**Review by TomasKostadinov — CHANGES_REQUESTED** [review]
More on this later, could not see it as it is currently not beeing deployed

**Review by BlueEnni — COMMENTED** [review]
Changed the destination path

**Review by TomasKostadinov — APPROVED** [review]
Seems fine ¯\_(ツ)_/¯

**Pull request merged**